config/betterquesting/DefaultQuests.json

```diff
@@ -317379,6 +317379,58 @@
         }
       },
       "rewards:9": {}
+    },
+    "3029:10": {
+      "questID:3": 3030,
+      "preRequisites:11": [
+        1197,
+        2668
+      ],
+      "properties:10": {
+        "betterquesting:10": {
+          "snd_complete:8": "random.levelup",
+          "taskLogic:8": "AND",
+          "visibility:8": "NORMAL",
+          "isMain:1": 0,
+          "simultaneous:1": 0,
+          "icon:10": {
+            "id:8": "gregtech:gt.blockmachines",
+            "Count:3": 1,
+            "Damage:2": 12791,
+            "OreDict:8": ""
+          },
+          "snd_update:8": "random.levelup",
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "OR",
+          "repeat_relative:1": 1,
+          "name:8": "§6§lHigh Temperature Gas-cooled Reactor",
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "§6§lHTGR§r§6 is a ZPM tiered reactor meant to be used as a breeder reactor to make Lutetium and gasses used in void miners.\n\n§rEficiency is calculated based on how full the reactor is, overall it works EXPONENTIALLY better when full.\nThe formula for efficiency is (fuel/72000)^2\nIt has minimal amount of fuel needed to start operation (10%).\nNeeds 1A LuV and 4000L of coolant(depending on fuel) to operate.\nFor more info check descriptions of items, the structure can be checked by blueprint.\n\nInformation about Fuels:\n\n90% of fuel casing parts are recyclable\n\n§2Thorium fuel is turned into Lutetium with 1:1 ratio (but 90% efficiency) also multiplies coolant needed by 1x\n§aUranium fuel is turned into Krypton also multiplies coolant needed by 1.5x\n§cPlutionium fuel is turned into Xenon also multiplies coolant needed by 2x\n\n§Y§9You can find all items by typing htgr in nei."
```

Comment:
Efficiency (typo) / 4000L of coolant per pebble, per tick, per second?

Reply:
4000L/s * (fuel multiplier) * (efficiency 1%-100%)

Reply:
Ok I specified it in the quest